pkg/cluster/clusterLeader.go

```diff
@@ -181,48 +193,70 @@ func (cluster *Cluster) StartLeaderCluster(c *kubevip.Config, sm *Manager) error
 					if err != nil {
 						log.Error(err)
 					}
-					projects, _, err := packetClient.Projects.List(nil)
-					if err != nil {
-						log.Error(err)
-					}
-					for _, p := range projects {
-						log.Println(p.ID, p.Name)
-
-						// Find our project
-						if p.Name == c.PacketProject {
-							ips, _, _ := packetClient.ProjectIPs.List(p.ID)
-							for _, ip := range ips {
 
-								// Find the device id for our EIP
-								if ip.Address == c.VIP {
-									log.Infof("Found EIP ->%s ID -> %s\n", ip.Address, ip.ID)
+					// We're using Packet with BGP
+					if c.EnableBGP {
+						log.Debugf("Looking up the BGP configuration from packet")
+						err = packet.BGPLookup(packetClient, c)
+						if err != nil {
+							log.Error(err)
+						}
+					} else {
 
```

Comment:
`// We're using Packet with ...` would compliment the previous comment

Reply:
This has reached a depth worth extracting it to a new function:
`err := packet.AssignEIP(id, c.PacketProject, c.VIP)`

Reply:
Moved the EIP function out to its own function, and simplified a lot of the code with helper functions. 👍